textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/neextractor/NamedEntityExtractorTest.java

```diff
@@ -0,0 +1,154 @@
+package edu.uci.ics.textdb.dataflow.neextractor;
+
+import edu.uci.ics.textdb.api.common.IField;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.common.Schema;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.api.dataflow.ISourceOperator;
+
+import edu.uci.ics.textdb.api.storage.IDataReader;
+import edu.uci.ics.textdb.api.storage.IDataStore;
+import edu.uci.ics.textdb.api.storage.IDataWriter;
+import edu.uci.ics.textdb.common.constants.LuceneConstants;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.TextField;
+import edu.uci.ics.textdb.dataflow.neextrator.NamedEntityExtractor;
+import edu.uci.ics.textdb.dataflow.source.ScanBasedSourceOperator;
+import edu.uci.ics.textdb.dataflow.utils.TestUtils;
+import edu.uci.ics.textdb.storage.LuceneDataStore;
+import edu.uci.ics.textdb.storage.reader.LuceneDataReader;
+import edu.uci.ics.textdb.storage.writer.LuceneDataWriter;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.Query;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Sam on 16/4/27.
+ */
+public class NamedEntityExtractorTest {
+    private NamedEntityExtractor neextractor;
+
+    private IDataWriter dataWriter;
+    private IDataReader dataReader;
+    private IDataStore dataStore;
+
+    private Query query;
+    private Analyzer analyzer;
+
+
+
+    @After
+    public void cleanUp() throws Exception {
+        dataWriter.clearData();
+    }
+
+
+
+    /**
+     * Scenario 1: Get next tuple with single return
+     * Text : Microsoft is a organization.
+     */
+    @Test
+    public void getNextTupleTest1() throws Exception {
+        List<ITuple> data = NEExtractorTestConstants.getTest1Tuples();
+        IOperator sourceOperator = TestHelper(data.get(0).getSchema(),data);
+
+        NamedEntityExtractor neExtractor = new NamedEntityExtractor(sourceOperator);
+        neExtractor.open();
+
+        ITuple iTuple=neExtractor.getNextTuple();
```

Comment:
General comment about formatting: Add " " before and after "=". Please use your IDE to format the files globally.